bundles/org.openhab.binding.shelly/README.md

```diff
@@ -374,7 +380,7 @@ A new alarm will be triggered on a new condition or every 5 minutes if the condi
 | TEMP_OVER  | Above "temperature over" threshold                                                               |
 | VIBRATION  | A vibration/tamper was detected (DW2 only)                                                       |
 
-Refer to section [Full Example:shelly.rules](#shellyrules) for examples how to catch alarm triggers in openHAB rules
+Refer to section Full Example for examples how to catch alarm triggers in openHAB rules
```

Comment:
```suggestion
Refer to section [Full Example](#full-example) for examples how to catch alarm triggers in openHAB rules.
```

Reply:
changed

Reply:
Are you sure the link will work? I'm no expert in Markdown, so I don't know if the dot will be part of the heading id? Also, do you really want it to show as "Full Example:shelly.rules", it seems a little compressed?

Reply:
Now noticed that you changed it back, so was easy to test on existing documentation page - the link is broken, but this link will work: https://www.openhab.org/addons/bindings/shelly/#shelly-rules

Reply:
@markus7017 Did you see this comment? It seems your changes are different. Can you comment on that?

Reply:
@markus7017 - now the link should work, but did you want a deep link to `#shelly-rules`? See suggestion above.

Reply:
Leaving it as is to reach M3.